src/Microsoft.ML.FastTree/FastTreeRanking.cs

```diff
@@ -456,7 +456,10 @@ protected override string GetTestGraphHeader()
         protected override RankingPredictionTransformer<FastTreeRankingModelParameters> MakeTransformer(FastTreeRankingModelParameters model, Schema trainSchema)
         => new RankingPredictionTransformer<FastTreeRankingModelParameters>(Host, model, trainSchema, FeatureColumn.Name);
 
-        public RankingPredictionTransformer<FastTreeRankingModelParameters> Train(IDataView trainData, IDataView validationData = null)
+        /// <summary>
+        /// Trains a <see cref="FastTreeRankingTrainer"/> using both training and validation data, returns a <see cref="ITransformer"/>.
```

Comment:
>ITransformer [](start = 120, length = 12)

Whenever possible, I'd choose a more specific type. Does `RankingPredictionTransformer<FastTreeRankingModelParameters>` look good to you?